dojo/tools/nikto/parser.py

```diff
@@ -43,26 +43,34 @@ def get_findings(self, filename, test):
 
     def process_json(self, file, test):
         data = json.load(file)
-
+        if len(data) == 1 and type(data) is list:
+            data = data[0]
         dupes = dict()
         host = data.get("host")
         port = data.get("port")
         if port is not None:
             port = int(port)
         for vulnerability in data.get("vulnerabilities", []):
+            if vulnerability.get('OSVDB') is None:
```

Comment:
Shouldn't this condition be flipped?
```suggestion
            if vulnerability.get('OSVDB') is not None:
```